config.toml

```diff
@@ -1,3 +1,6 @@
-venv_path = "./venv/bin/activate"
-workspace_path = "./ocrd-workspace/"
-mets_path = "./ocrd-workspace/mets.xml"
\ No newline at end of file
+# See https://www.nextflow.io/docs/latest/process.html#script for the difference
+# between NextFlow and Bash variables.
+venv_path = "\\$HOME/venv37-ocrd/bin/activate"
```

Comment:
A single escape character is needed before $HOME, not two.

Reply:
In TOML `\$` is not an allowed control sequence.

If you load the config file with `tomli`, it resolves correctly:

```python
import tomli

with open('config.toml', mode='rb') as fp:
    config = tomli.load(fp)

print(config['venv_path']
```

This yields `\$HOME/venv37-ocrd/bin/activate`. That should be correct, right?